### PR TITLE
Set mutualized print by default

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/+dot+env.mako_tmpl [hunk 1]
```diff
@@ -29,7 +29,7 @@ TILEGENERATION_SQS_QUEUE=<queue_name>
 GUNICORN_PARAMS=--bind=:8080 --worker-class=gthread --threads=10 --workers=5 --timeout=60 --max-requests=1000 --max-requests-jitter=100
 TINYOWS_URL=http://tinyows:8080/
 MAPSERVER_URL=http://mapserver:8080/
-PRINT_URL=http://print:8080/print/
+PRINT_URL=https://mutualized-print.paas-ch-3.camptocamp.com/print/demo_geomapfish_24/
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/+dot+env.mako_tmpl]: Are you sure you want to hardcode the demo_geomapfish_24 app?

**Comment by gberaudo** [comment]
Why this change?
I think a new c2cgeoportal application should have a working print set up by default.

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/+dot+env.mako_tmpl (hunk 1 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/+dot+env.mako_tmpl]: Thanks, fixed :-)

**Comment by sbrunner** [comment]
We want to promote the mutualized print, and as yous see by modifying about 10 lines we get the internal print back :-)

File: doc/integrator/print.rst [hunk 2]
```diff
@@ -19,72 +16,34 @@ The main configuration file is ``print/print-apps/<package>/config.yaml``.
...
-
+Infrastructure
+--------------
 
-Having a dedicated print instance
----------------------------------
+By default we use the mutualized print Platform as a service from Camptocamp.
 
-The goal is to be able to create a custom makefile with which one we make:
+To configure it you should ask Camptocamp for create e.-g. tow new applications on the platform, one for the integration,
```
Comment (pvalsecc) [inline doc/integrator/print.rst]:
```suggestion
To configure it, you should ask Camptocamp for create e.-g. two new applications on the platform, one for the integration,
```

File: doc/integrator/print.rst [hunk 3]
```diff
@@ -19,72 +16,34 @@ The main configuration file is ``print/print-apps/<package>/config.yaml``.
...
+One for the production.
```
Comment (pvalsecc) [inline doc/integrator/print.rst]:
```suggestion
one for the production.
```

File: doc/integrator/print.rst [hunk 4]
```diff
@@ -19,72 +16,34 @@ The main configuration file is ``print/print-apps/<package>/config.yaml``.
...
 
-.. prompt:: bash
+Configure the application name in the ``MUTUALIZED_PRINT_APP`` variable of a ``Makefile``.
 
-   ./docker-run make --makefile=<file>.mk
+Devlopement
```
Comment (pvalsecc) [inline doc/integrator/print.rst]:
```suggestion
Development
```

File: doc/integrator/print.rst [hunk 5]
```diff
@@ -19,72 +16,34 @@ The main configuration file is ``print/print-apps/<package>/config.yaml``.
...
+Internal print
+--------------
 
-.. code:: yaml
+To use the internal print you should:
 
-   BUILD_RULES = test-packages print
-   TEST_PACKAGES = main print
+* Set ``PRINT_URL`` to ``http://print:8080/print/`` in the ``.env.mako`` file.
+* Uncommant the ``print`` service in the ``docker-compose.yaml`` file.
```
Comment (pvalsecc) [inline doc/integrator/print.rst]:
```suggestion
* Uncomment the ``print`` service in the ``docker-compose.yaml`` file.
```

File: doc/integrator/print.rst [hunk 6]
```diff
@@ -19,72 +16,34 @@ The main configuration file is ``print/print-apps/<package>/config.yaml``.
...
-
+Infrastructure
+--------------
 
-Having a dedicated print instance
----------------------------------
+By default we use the mutualized print Platform as a service from Camptocamp.
 
-The goal is to be able to create a custom makefile with which one we make:
+To configure it, you should ask Camptocamp for create e.-g. two new applications on the platform, one for the integration,
```
Comment (jwkaltz) [inline doc/integrator/print.rst]: for create -> to create

**Pull request merged**